server.go

```diff
@@ -5314,7 +5314,7 @@ func (s *server) applyChannelUpdate(update *lnwire.ChannelUpdate1,
 func (s *server) SendCustomMessage(peerPub [33]byte, msgType lnwire.MessageType,
 	data []byte) error {
 
-	peer, err := s.FindPeerByPubStr(hex.EncodeToString(peerPub[:]))
+	peer, err := s.FindPeerByPubStr(string(peerPub[:]))
```

Comment:
we should fix this in the `FindPeerByPubStr` method, we should use a proper hex-encoded string.